pkg/cmd/server/kubernetes/master.go

```diff
@@ -67,6 +68,8 @@ const (
 // endpoints were started (these are format strings that will expect to be sent
 // a single string value).
 func (c *MasterConfig) InstallAPI(container *restful.Container) ([]string, error) {
+	// Wait for the next rebase
+	// c.Master.StorageFactory.AddCohabitatingResources(extensions.Resource("replicasets"), kapi.Resource("replicationcontrollers"))
```

Comment:
@deads2k so with this in place, the interface returned by `storage.New(extensions.Resource("replicasets"))` should be enough for plugging into the custom rc reststorage? I still don't understand how the custom storage is supposed to delegate.